cl/search/management/commands/pacer_bulk_fetch.py

```diff
@@ -0,0 +1,385 @@
+
+        if remaining_courts[court_id]:
+            doc = remaining_courts[court_id].pop(0)
+            try:
+                fq = self.enqueue_pacer_fetch(doc)
+                self.update_fetches_in_progress(court_id, fq.id)
+            except Exception as e:
+                self.total_errors += 1
+                logger.error(
+                    f"Error queuing document {doc.get('id')}: {str(e)}"
+                )
+
+        return False
+
+    def fetch_docs_from_pacer(self) -> None:
+        """Process documents with one fetch per court at a time"""
+        self.handle_pacer_session()
+        remaining_courts = self.courts_with_docs.copy()
+
+        while remaining_courts:
+            courts_at_start = len(remaining_courts)
+            skipped_courts = 0
+            for court_id in list(remaining_courts.keys()):
+                was_skipped = self.fetch_next_doc_in_court(
+                    court_id,
+                    remaining_courts,
+                )
+                skipped_courts += int(was_skipped)
+                # If this court doesn't have any more docs, remove from dict:
+                if not remaining_courts[court_id]:
+                    remaining_courts.pop(court_id)
+                    self.fetches_in_progress.pop(court_id, None)
+
+            # If we had to skip all courts that we tried this round,
+            # add a small delay to avoid hammering the DB
+            if skipped_courts == courts_at_start:
+                time.sleep(self.interval)
+
+    def process_docs_fetched(self):
+        """Apply tasks to process docs that were successfully fetched from PACER."""
+        cached_fetches = cache.get(self.docs_to_process_cache_key(), [])
+        fetch_queues_to_process = [fq_pk for (_, fq_pk) in cached_fetches]
+        fetch_queues = (
+            PacerFetchQueue.objects.filter(pk__in=fetch_queues_to_process)
+            .select_related("recap_document")
+            .only(
+                "pk",
+                "status",
+                "recap_document__id",
+                "recap_document__ocr_status",
+                "recap_document__is_available",
+                "recap_document__filepath_local",
+            )
+        )
+        for fq in fetch_queues:
+            rd = fq.recap_document
+            needs_ocr = rd.needs_extraction
+            has_pdf = rd.filepath_local is not None
+            fetch_was_successful = fq.status == PROCESSING_STATUS.SUCCESSFUL
+            if fetch_was_successful and has_pdf and needs_ocr:
+                self.throttle.maybe_wait()
+                chain(
+                    extract_recap_pdf.si(rd.pk),
+                    mark_fq_successful.si(fq.pk),
+                ).apply_async(queue=self.queue_name)
+
+    def handle_fetch_docs(self):
+        """Run only the fetching stage."""
+        logger.info("Starting fetch stage in pacer_bulk_fetch command.")
+        try:
+            self.set_user(self.options.get("username", "recap"))
```

Comment:
Again, the default should come from argparse.

Reply:
solved now defaults to `recap`